aiohttp/client.py

```diff
@@ -167,6 +173,75 @@ def request(self, method, url, *,
 
         return resp
 
+    @asyncio.coroutine
+    def ws_connect(self, url,
+                   protocols=(),
+                   timeout=10.0,
+                   connector=None,
+                   response_class=None,
```

Comment:
I think better to invite new name for parameter: *response_class* clashes with parameter with the same name from constructor.